taichi/ir/type.h

```diff
@@ -175,6 +175,8 @@ class TensorType : public Type {
     return shape_;
   }
 
+  int vector_width() const;
```

Comment:
mark `override`? (overrides Type::vector_width())

Reply:
Nice catch. Just realized that this is no longer used. (this was used by LocalLoadStmt for loading multiple values, but now we are wrapping into a `MatrixInitStmt`)